### DISCUSSION TITLE
Vitocal 200-S + 2x Vitoair FS 300E MA per CAN ?

### DISCUSSION THREAD

**Discussion by Perseus90** [open3e/open3e#135] [Q&A] [answered]
Hallo zusammen,

weiß jemand ob es möglich wäre eine Vitocal 200-S + 2x Vitoair FS 300E MA über den CAN Bus auszulesen und zu steuern?
Ich finde leider nichts direkt dazu, da Viessmann sich da sehr bedeckt zu hält.

Hintergrund: Ich plane gerade mein Smarthome und die Heizung/Lüftung über eine App zu Steuern (Cloud +Abo incl.) ist für mich nicht gerade smart.

Ich hoffe ihr könnt mir weiterhelfen.

Grüße
-Percy

**Comment by philippoo66** [comment]
hi @Perseus90 
erstmal sorry für die späte Reaktion. Sind denn die beiden Vairs schon mit der Vitocal gekoppelt? Wenn ja sollte das kein Problem sein, auf alles per CAN/open3e zuzugreifen, wobei die VAir nicht wirklich viel bietet soweit ich erinnere. @DpunktS betreibt Vcal und Vair, der kann dir bestimmt mehr sagen.
Grüsse!
Phil

ps. falls dein Smarthome ioBroker hat, nimm gleich den Adapter.

**Reply by DpunktS** [reply]
Hallo @Perseus90,

ja ich betreibe wie @philippoo66 schon geschrieben hat eine Vitocal 250-a und eine Vitoair FS 300E mit dem Iobroker Adapter. Ich sehe alle relevanten Sensorwerte der Vitoair und kann sie auch problemlos über den Adapter bedienen.

**Reply by Perseus90** [reply]
Danke schonmal für eure Antworten :)
Ich bin aktuell noch in der Planungsphase und habe leider nichts da um es einfach mal auszuprobieren.

Ist es auch möglich 2 Vitoair am CAN Bus zu haben? Nicht das da irgendwelche Hardcodierten/Typcodierten Werte sich überschneiden oder so?
Ich habe geplant KNX im Haus zu verbauen und würde "Spielereien" per Node-Red auf einem Raspy machen, dort würde ich dann auch open3e laufen lassen. Geht der ioBroker Adapter über die Viessmann API also Cloud und Kosten?

Wäre es alternativ auch möglich alle 3 Anlagen über die APs auszuwerten?

**Reply by MyHomeMyData** [reply]
> Geht der ioBroker Adapter über die Viessmann API also Cloud und Kosten?

Der Adapter [e3oncan](https://github.com/MyHomeMyData/ioBroker.e3oncan#readme) holt sich die Daten auch über den CAN-Bus, genauso wie open3e. Hat aber ein paar zusätzliche Features, z.B. werden auch Energy Meter unterstützt.

**Reply by philippoo66** [reply, marked as answer]
@Perseus90 
> Ist es auch möglich 2 Vitoair am CAN Bus zu haben? Nicht das da irgendwelche Hardcodierten/Typcodierten Werte sich überschneiden oder so?

das ist kein Problem. die bekommen unterschiedliche 'IDs'. du muss beim Einbau der Anlage aber darauf achten/bestehen, dass die Geräte gleich CAN-mässig verbunden werden. Das muss nämlich (u.a. wegen der ID 'Vergabe') explizit eingerichtet (oder so) werden, das sparen sich die HBs gern...

**Reply by philippoo66** [reply]
> Wäre es alternativ auch möglich alle 3 Anlagen über die APs auszuwerten?

haben die Vairs überhaupt sowas? Gewöhnich schaltet sich der AP Modus nach einer Weile ab, ich meine aber zu erinnern, dass es irgendwer mal durch irgendwelche regelmässigen Aktionen geschafft hätte, den dauerhaft aktiv zu halten. Ich würde das aber nicht empfehlen. Die CAN Geschichte ist einfach 'robuster' und unproblematischer (wenn einmal alles verbunden ist).

**Reply by Perseus90** [reply]
@philippoo66 vielen Dank für die Entscheidungshilfe
